templates/desktop-menu.twig

```diff
@@ -13,6 +13,7 @@
                     <a
                             class="nav-link"
                             href="{{ item.get_link }}"
+                            target={{ item.target }}
```

Comment:
I think it needs to be enclosed in double quotes, same as you did for submenu.

![image](https://github.com/greenpeace/planet4-master-theme/assets/939357/0616f98e-d927-4b58-9ffe-389314ffeee4)